### ISSUE TITLE
[Feature Request] Log unwanted input metrics .

### ISSUE THREAD

**Issue by toni-moreno** [grobian/carbon-c-relay#78]
Could be very useful if we can get a log to audit which metrics  and  the IP sources of unwanted metrics

something like

```
match  "^caseA"  send cluster_A stop;
match "^caseB" send cluster_B stop;
match "^caseC" send cluster_C stop;
match "*" send blackhole[ /var/log/carbon-c-relay/unwanted.log] stop;
```

This log should have at least IP origin and metric name.

**Comment by grobian** [comment]
perhaps a special cluster type for this would work too?

cluster log file /var/log/wherever.log;
match * send to log;

**Comment by toni-moreno** [comment]
Yes @grobian this would  be great to me.!!

When  do you think it would be implemented?

**Comment by grobian** [comment]
I need to find some time, which I can't make any guarantees of at the momement.  Hope that's ok.

**Comment by toni-moreno** [comment]
That's ok.

Thank you very much !!

**Comment by grobian** [comment]
this is a bit more work than I anticipated :) anyway, working on it from the garden now :D

**Comment by grobian** [comment]
OK, got file-logging implemented.  IP logging is a small challenge, for that information is not available during log time.

**Comment by grobian** [comment]
would you be very disappointed if IP logging was not available?

**Comment by toni-moreno** [comment]
Ummmm... I can understand the challenge, and of course  l'll be happy to use carbon-c-relay anyway.

 There is any other way to detect who ( ip source) are sending unwanted metrics ?

**Comment by grobian** [comment]
Ok, then I'll keep this request open.

**Comment by grobian** [comment]
source ip implemented in 7d3c3c5315803bfcd21704589999287eee1015bd

**Comment by toni-moreno** [comment]
Hi  @grobian . 

I'm really happy to see this Feature Request full done !!

I will try to give you feedback as soon as i could compile and test it.

Lots of thanks !!!
